acceptance-test/src/acceptance-test/java/org/zalando/nakadi/repository/db/SubscriptionDbRepositoryTest.java

```diff
@@ -170,7 +173,7 @@ public void whenListSubscriptionsLimitAndOffsetAreRespected() throws ServiceTemp
         testSubscriptions.subList(3, testSubscriptions.size()).clear();
 
         final List<Subscription> subscriptions = repository.listSubscriptions(
-                emptySet(), Optional.of(owningApp), 2, 3);
+                emptySet(), Optional.of(owningApp), Optional.empty(), 2, 3);
```

Comment:
All tests here are using empty value for the newly introduced parameter.  Do we want an actual test that passes some reader filter?